core/src/impl/Kokkos_ConcurrentBitset.hpp

```diff
@@ -249,6 +249,10 @@ struct concurrent_bitset {
       if (!(prev & mask)) {
         // Successfully claimed 'result.first' by
         // atomically setting that bit.
+        // Flush the set operation. Technically this only needs to be acquire/
+        // release semantics and not sequentially consistent, but for now
+        // we'll just do this.
```

Comment:
Perhaps make this a TODO/issue? If we ever implement those semantics, it would be nice to be able to quickly implement these changes